src/components/Modals/Send/views/Address.tsx

```diff
@@ -41,9 +41,6 @@ export const Address = () => {
   const asset = useWatch<SendInput, SendFormFields.Asset>({ name: SendFormFields.Asset })
   const isYatFeatureEnabled = useFeatureFlag('Yat')
 
-  // reset address when asset changes
-  useEffect(() => setValue(SendFormFields.Input, ''), [asset, setValue])
-
```

Comment:
This was wrong because it was trigger every time the component was mounted, meaning that if you scan a QR code, when you get back to the address input, it was clearing it. Better clearing it when you select the asset directly